apps/right-to-rent-check/translations/src/en/fields.json

```diff
@@ -1,5 +1,33 @@
 {
-  "name": {
-    "label": "Name"
+  "living-status": {
+    "options": {
+      "yes": {
+        "label": "Yes"
+      },
+      "no": {
+        "label": "No"
+      }
+    }
+  },
+  "tenant-name": {
+    "label": "Prospective tenant's full name"
+  },
+  "tenant-dob": {
+    "legend": "Date of Birth",
+    "hint": "For example, 12 3 1980",
+    "parts": {
+      "day": {
+        "label": "Day"
+      },
+      "month": {
+        "label": "Month"
+      },
+      "year": {
+        "label": "Year"
+      }
+    }
```

Comment:
it was intentional in bootstrap, as you _may_ want to include custom ones, especially if including a different language. I don't think another example is _needed_ I don't want the example to be too convoluted, it should be in the readme and it _will_ be added to the bootstrap docs once we get round to them